examples/hd44780_i2c/README.md

```diff
@@ -0,0 +1,40 @@
+# Example for `hd44780` and `pcf8574` driver
+
+An example to use `HD44780` LCD with `PCF8574` Remote 8-Bit I/O Expander for
+I2C Bus.
+
+## What it does
+
+It prints "Hello world!" with two user-defined characters.
+
+It prints current time in a loop.
+
+## Wiring
+
+In most cases, the `HD44780` module comes with a `PCF8574` module designed for
+`HD44780`. The `PCF8574` module has pins for I2C and `hd44780`. In that case,
+you simply connect two modules. The example assumes a typical configuration,
+described below.
+
+| `hd44780` | `PCF8574` |
+|-----------|-----------|
+| `RS`      | 0         |
+| `E`       | 1         |
```

Comment:
i do not have a PCF8574 module at hand, but this should be `2`. users who use bare PCF8574 can easily modify wiring, but those who use a PCF8574 module cannot.

Reply:
It is 2:
```C
    hd44780_t lcd = {
        .write_cb = write_lcd_data, // use callback to send data to LCD by I2C GPIO expander
        .font = HD44780_FONT_5X8,
        .lines = 2,
        .pins = {
            .rs = 0,
            .e  = 2,
            .d4 = 4,
            .d5 = 5,
            .d6 = 6,
            .d7 = 7,
            .bl = 3
        }
    };
```

Reply:
you are right. not enough coffee that day.